internal/pkg/template/templates/workloads/services/rd-web/cf.yml

```diff
@@ -133,12 +133,18 @@ Resources:
         HealthyThreshold: !If [HasHealthCheckHealthyThreshold, !Ref HealthCheckHealthyThreshold, !Ref AWS::NoValue]
         UnhealthyThreshold: !If [HasHealthCheckUnhealthyThreshold, !Ref HealthCheckUnhealthyThreshold, !Ref AWS::NoValue]
       {{- end }}
-      {{- if requiresVPCConnector .}}
       NetworkConfiguration:
         EgressConfiguration:
+          {{- if requiresVPCConnector .}}
           EgressType: VPC
           VpcConnectorArn: !Ref VpcConnector
-      {{- end }}
+          {{- else }}
+          EgressType: DEFAULT
```

Comment:
Is this one of those field that you can't remove after adding it? What does it mean if a template has `EgressType: DEFAULT`, and then it's deployed again without that field?

A more realistic example: say Copilot adds `EgressType: Default` into the template, and then the deployment fails, triggering a rollback removing the field. Will the CFN handler complain?